be/src/storage/lake/txn_log_applier.cpp

```diff
@@ -148,6 +151,22 @@ class PrimaryKeyTxnLogApplier : public TxnLogApplier {
         return Status::OK();
     }
 
+    Status apply_alter_meta_log(const TxnLogPB_OpAlterMeta& op_alter_meta) {
+        DCHECK_EQ(_base_version + 1, _new_version);
+        switch (op_alter_meta.table_meta_type()) {
+        case TabletMetaTypePB::ENABLE_PERSISTENT_INDEX:
+            _metadata->set_enable_persistent_index(op_alter_meta.enable_persistent_index());
```

Comment:
check if current `enable_persistent_index` is same with `op_alter_meta.enable_persistent_index`, if same, try not to remove index?

Reply:
FE will check this，If they are the same, FE will not send the job to BE for modification。but ok，just in case, I will also add a check logic to BE.